### ISSUE TITLE
draft - Principle 8 - Don’t forget accessibility!

### ISSUE THREAD

**Issue by ToniBonittoGSA** [GSA/digitalgov.gov#911]
Principle 8 - Don’t forget accessibility! 

## Welcome to Principle 8, our last principle!

Most of us can see, hear, and touch our mobile devices as they were designed, but many others are not. In the federal government, our customers are everyone—and we have a duty to help everyone have a great experience on our websites and mobile apps. 

This week we will focus on accessibility, the lack of which leads to mobile unfriendliness. The accessibility of federal websites is covered in the [Information Technology & Innovation Foundation (ITIF) Benchmarking Report](https://itif.org/publications/2017/03/08/benchmarking-us-government-websites).

**Issue: Developers are not capitalizing on the accessibility features available on mobile devices**  

As we develop our sites, we build them for their visual appeal—but often forget that there are many visually-impaired users. To varying degrees, these users may not be able to see or navigate our websites on desktop or mobile devices since they do not use a mouse while navigating the Internet. 

Some specific challenges the visually impaired have are related to poor color contrast and the lack of labels, which may hinder their screen reader. To prevent this, assistive technology (AT) must be used.

**Solution: Build a site that is accessible to everyone using The Web Content Accessibility Guidelines (WCAG)!**

Accessible sites are designed to reduce—if not eliminate—the barriers that prevent people with disabilities from using them. At the most general level, accessible sites adhere to accessible design principles, including:

- High-contrast colors (for text, borders, backgrounds)
- Alternate text (“alt text”) that describes images for blind or low-vision users
- Labels for buttons, so the screen reader can help to navigate the site 
- Captions and or transcripts for videos and podcasts. 

In 2008, the [World Wide Web Consortium](https://www.w3.org/WAI/standards-guidelines/wcag/) (W3C) published the Web Content Accessibility Guidelines (WCAG; pronounced: “way-cagg” or “wuh-cog”) to specify how web developers should make content accessible for people with disabilities. The WCAG 2.0 has Success Criteria (SC) split between three [Levels of Conformance](https://www.w3.org/TR/UNDERSTANDING-WCAG20/conformance.html) (low to high): A, AA, and AAA; with AAA being the level that makes your content the most accessible. 

In January 2017, the [United States Access Board](https://www.access-board.gov/) (also known as the Architectural and Transportation Barriers Compliance Board) adopted final rules to make [WCAG 2.0 Level AA](https://www.w3.org/WAI/WCAG21/quickref/?currentsidebar=%23col_overview&versions=2.0&levels=a%2Caaa#top) the accessibility standards that the federal government uses to provide accessible web services. The WCAG 2.0 guidelines outline techniques to optimize content and are based on the following four “POUR” principles, where all online content should be: 

1. **Perceivable** - Information and user interface components must be presentable to users in ways they can perceive.
2. **Operable** - User interface components and navigation must be operable.
3. **Understandable** - Information and the operation of user interface must be understandable.
4. **Robust** - Content must be robust enough that it can be interpreted reliably by a wide variety of user agents, including assistive technology.

Details on the four “POUR” principles can be found here: [https://www.w3.org/TR/UNDERSTANDING-WCAG20/intro.html#introduction-fourprincs-head](https://www.w3.org/TR/UNDERSTANDING-WCAG20/intro.html#introduction-fourprincs-head) 

To test for accessibility compliance, the free [AChecker Web Accessibility Checker](https://achecker.ca/checker/) is available and checks against the WCAG 2.0’s Level AA Conformance (see Section508.gov’s [Applicability & Conformance Requirements](https://www.section508.gov/create/applicability-conformance) for more information). 

## References

- Akamai Reveals 2 Seconds as The New Threshold of Accessibility for e-Commerce Web Page Response Times | https://www.akamai.com/us/en/about/news/press/2009-press/akamai-reveals-2-seconds-as-the-new-threshold-of-acceptability-for-ecommerce-web-page-responsetimes.jsp
- Improving Website Accessibility | https://www.section508.gov/content/build/websiteaccessibility-improvement
- WCAG 2.0 AA Gains Prominence as Website Accessibility Standard | http://www.natlawreview.com/article/wcag-20-aa-gains-prominencewebsite-accessibility-standard
- Web Accessibility Checker | http://achecker.ca/checker/
- Web Accessibility Checker Documentation | https://achecker.ca/documentation/index.php?p=checker/index.php 
- Web Content Accessibility Guidelines (WCAG) 2.0 | https://www.w3.org/TR/WCAG20/ 

## Additional Resources

- 18F Accessibility Guide | https://accessibility.18f.gov/ 
- U.S. Access Board | https://www.access-board.gov/
- Training | https://www.access-board.gov/training/webinars 
- Accessibility for Teams | https://accessibility.digital.gov/ 
- Americans with Disabilities Act (ADA) | https://www.ada.gov/ 
- How to Meet WCAG 2.0 Quick Reference | https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0 
- Plain Language | https://www.plainlanguage.gov/ 
- Section 508 | https://www.section508.gov/ 
- Revised 508 Standards Refresh Toolkit | https://www.section508.gov/refresh-toolkit 
- Create Accessible Digital Products | https://www.section508.gov/create 
- Training | https://www.section508.gov/content/training 
- Usability | https://www.usability.gov/ 
- Web Accessibility Initiative (WAI) - https://www.w3.org/WAI/
- Web Accessibility Perspectives: Explore the Impact and Benefits for Everyone - https://www.w3.org/WAI/perspectives/ 
- Web Accessibility Tutorials and Presentations - https://www.w3.org/WAI/train 

--- 

_All references to specific brands, products, and/or companies are used only for illustrative purposes and do not imply endorsement by the U.S. federal government or any federal government agency._
